apps/website/components/README.md

```diff
@@ -1 +1,3 @@
 # Component Overview
+
+<ComponentsOverview :componentList="$site.themeConfig.sidebar.find(x => x.title === 'Components').children" />
```

Comment:
I would prefer to have the function/logic in a computed property so that the template stays simple.

Reply:
I was also thinking that this would have a more general naame, e.g:

```<DocItemsOverview :items="'Components">```

That way it could easily be re-used for 

```<DocItemsOverview :items="'Foundations">```

Reply:
@hippee-lee  The thing with using the plural for me is that it is casual to misread the name and think of it as a single element. When you have the `List` in the end - there is no way you would think this component is responsible to display only a single item. Let me know what do you think.